sycl/doc/SYCLEnvironmentVariables.md

```diff
@@ -4,11 +4,19 @@ This file describes environment variables that are having effect on SYCL compile
 
 # Controlling SYCL RT
 
-| Environment variable | Description |
-| ----------- | ----------- |
-| SYCL_PI_TRACE | If set forces tracing of PI calls to stdout. |
-| SYCL_BE={PI_OPENCL,PI_OTHER} | When SYCL RT is built with PI this controls which plugin to use. |
-| SYCL_PRINT_EXECUTION_GRAPH | Print execution graph to DOT text file. Options are described below. |
+| Environment variable | Values | Description |
+| -------------------- | ------ | ----------- |
+| SYCL_PI_TRACE | Any(*) | Force tracing of PI calls to stderr. |
+| SYCL_BE | PI_OPENCL, PI_OTHER | When SYCL RT is built with PI this controls which plugin to use. Default value is PI_OPENCL. |
+| SYCL_DEVICE_TYPE | One of: CPU, GPU, ACC, HOST | Force SYCL to use specified device type. If unset, default selection rules are applied. If set to other value, this control has no effect. If the requested device type is not found, `cl::sycl::runtime_error` exception is thrown. If non-default device selector is used, device must satisfy both selector and this control. |
+| SYCL_PROGRAM_BUILD_OPTIONS | String of valid OpenCL build options | Override build options for all programs. |
+| SYCL_USE_KERNEL_SPV | Path to SPIR-V binary | Load device image from specified file. If runtime is unable to read specified file, `cl::sycl::runtime_error` exception is thrown.|
+| SYCL_DUMP_IMAGES | Any(*) | Dump device image binaries to file. Control has no effect if SYCL_USE_KERNEL_SPV is set. |
```

Comment:
Is there no way to control the output file path?

Reply:
@AGindinson nope
https://github.com/intel/llvm/blob/78d9957e23f805d588c39531af4a8e27854d9314/sycl/source/detail/program_manager/program_manager.cpp#L386-L399